### PR TITLE
Function cmd

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- Add two new root commands: `apply` and `init` 
- Add `function` subcommand to both new commands

**Related issue(s)**

[Issue](https://github.com/kyma-project/cli/issues/551)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -34,13 +37,9 @@ require (
...
 	github.com/spf13/cobra v1.0.0
 	github.com/stretchr/testify v1.6.1
-	go.opencensus.io v0.22.4 // indirect
-	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208 // indirect
-	golang.org/x/sys v0.0.0-20200722175500-76b94024e4b6 // indirect
-	google.golang.org/grpc v1.29.1 // indirect
-	google.golang.org/protobuf v1.25.0 // indirect
 	gopkg.in/src-d/go-git.v4 v4.13.1
 	gopkg.in/yaml.v2 v2.3.0
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
```
Comment (clebs) [inline go.mod:42]: Please try to import only 1 version of this package if possible.

File: cmd/kyma/apply/apply.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+//NewCmd creates a new function command
+func NewCmd(o *cli.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "apply",
+		Short: "Applies local resources to the Kyma cluster.",
+		Long:  "Use this command to apply the local files and folders from your project to the Kyma cluster.",
```
Comment (clebs) [inline cmd/kyma/apply/apply.go:14]: This description needs to be more generic. In the future we will have other apply commands that will not necessarily apply files and folders from a project.

Comment (pPrecel) [reply cmd/kyma/apply/apply.go:14]: ok, good point

**Comment by pPrecel** [comment]
New PR: https://github.com/kyma-project/cli/pull/606

**Pull request closed without merging**